src/components/Layout/HomeContent.js

```diff
@@ -131,100 +131,100 @@ export function HomeContent() {
             React
           </h1>
           <p className="text-4xl font-display max-w-lg md:max-w-full py-1 text-center text-secondary dark:text-primary-dark leading-snug self-center">
-            The library for web and native user interfaces
+            A biblioteca para web e interfaces de usuário nativas
           </p>
           <div className="mt-5 self-center flex gap-2 w-full sm:w-auto flex-col sm:flex-row">
             <ButtonLink
               href={'/learn'}
               type="primary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="Learn React">
-              Learn React
+              label="Aprenda React">
+              Aprenda React
             </ButtonLink>
             <ButtonLink
               href={'/reference/react'}
               type="secondary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="API Reference">
-              API Reference
+              label="API Referência">
+              API Referência
```

Comment:
```suggestion
              Referência das APIs
```

O texto é meio grande para o botão, mas acho que seria mais adequado. Será que fica ruim no mobile?

Reply:
Vou testar